pkg/talking_leds/doc.txt

```diff
@@ -0,0 +1,6 @@
+/**
+ * @defgroup pkg_talking_leds Talking LEDs - Ardunio library for demonstration
```

Comment:
Typo: s/Ardunio/Arduino/

Here and below (in `@brief`)

Reply:
Fixed.